comm/src/main/scala/coop/rchain/comm/network.scala

```diff
@@ -42,18 +40,13 @@ class UnicastNetwork(peer: PeerNode, next: Option[ProtocolDispatcher[SocketAddre
   val comm  = new UnicastComm(local)
   val table = PeerTable(local)
 
-  def receiver[
-      F[_]: Monad: Capture: Log: Time: Metrics: TransportLayer: NodeDiscovery: Encryption: Kvs[
-        ?[_],
-        PeerNode,
-        Array[Byte]]: ApplicativeError_[?[_], CommError]: PacketHandler]: F[Unit] =
+  def receiver[F[_]: Monad: Capture: Log: Time: Metrics](
+      implicit protocolDispatcher: ProtocolDispatcher[F, SocketAddress]): F[Unit] =
```

Comment:
Maybe you can create a type alias to get rid of the `implicit protocolDispatcher: ProtocolDispatcher[F, SocketAddress]`?

Reply:
ProtocolDispatcher will be removed in next PR :) [as you already know :) ]